mkdocs.yml

```diff
@@ -50,8 +51,25 @@ nav:
       - 'Inclusion': 'ProgramOverview/inclusion.md'
       - 'Payment': 'ProgramOverview/payment.md'
       - 'Evaluation': 'ProgramOverview/evaluation.md'
+    - 'Users and Roles' :
+      - 'Users page': 'general/users-permissions.md'
+      - 'Roles-based permissions': 'general/table-description-role-permissions.md'
     - 'Program Team': 
       - 'Team page': 'general/team-page.md'
       - 'Add team members': 'general/add-team-members.md'
       - 'Edit team members roles': 'general/edit-team-roles-program.md'
       - 'Remove team members': 'general/remove-team-members-program.md'
+    - 'Program Design':
+      - 'Program Design': 'ProgramOverview/design.md'
+      - 'Read & Change Program Design': 'ProgramOverview/read-change-design-details.md'
+    - 'Registration':
+      - 'Registration page': 'ProgramOverview/registration.md'
```

Comment:
I would not call it 'Registration page', Maybe 'Registrations overview'?

Reply:
Yes, that's fine to me. to be more consistent, I will do the same for the other page, as Team page = Team Overview ? etc.